.prettierrc.js

```diff
@@ -0,0 +1,15 @@
+// Documentation for this file: https://prettier.io/docs/en/configuration.html
+module.exports = {
+  // Use .gitattributes to manage newlines
+  endOfLine: "auto",
+
+  // Avoid line breaks in markdown
+  proseWrap: "never",
+
+  // Prettier plugins
+  //
+  // Disabling search dirs and pointing directly at folder paths relative to this config file
+  // makes it easier for VSCode and WebStorm to find your plugins.
```

Comment:
interesting - do you happen to have a link to some issues related to this? I'm just curious why the automatic plugin search tends to fail in those contexts

Reply:
Now that you've challenged me on it, I suspect that this is a defense mechanism for Rush in particular.

If you have a Lerna repo or something and you use a root-level `package.json` to keep your prettier devDependency, which lives in a root-level `node_modules`, then I think 99.9% of the time VSCode would "do the right thing".

But in a Rush repo, there is no root package or node modules, and plugins are kept in a special folder; VSCode typically can't find this folder unless you specify exactly where it is. I can test out simplifying this for this repo since it probably would work just fine.

Reply:
Ah, ye - this might makes sense in the context of Rush if the location of plugins etc is not standard. That being said, I **prefer** an explicit list of plugins rather than one based on auto-search.